tests/test_sampling_exceptions.py

```diff
@@ -29,7 +29,7 @@ async def test_sampling_initialization_failed(api_url: str) -> None:
     operation = await helpers.sample(api_url, program_code, payload)
 
     # verify an error occurred
-    assert operation["result"].get("code") == 400
+    assert operation["result"].get("code") == 400, operation["result"]
```

Comment:
will this work?

Is this same as 

    assert bool, bool

    assert True, False --> assertion is not raised